--- origsrc/QScintilla-gpl-2.8.4/Python/configure.py	2014-09-11 12:15:17.000000000 -0500
+++ src/QScintilla-gpl-2.8.4/Python/configure.py	2014-10-30 02:40:07.243633400 -0500
@@ -1355,13 +1355,17 @@ win32 {
     PY_MODULE = %s.pyd
     target.files = %s.pyd
     LIBS += -L%s
+} else:cygwin-* {
+    PY_MODULE = %s.dll
+    target.files = %s.dll
+    LIBS_PRIVATE += -lpython%d.%d%s
 } else {
     PY_MODULE = %s.so
     target.files = %s.so
 }
 
 target.CONFIG = no_check_exist
-''' % (mname, mname, quote(target_config.py_pylib_dir), mname, mname))
+''' % (mname, mname, quote(target_config.py_pylib_dir), mname, mname, target_config.py_version >> 16, (target_config.py_version >> 8) &0x0ff, getattr(sys, 'abiflags', ''), mname, mname))
 
     pro.write('''
 target.path = %s
