--- origsrc/QScintilla-gpl-2.8.4/Qt4Qt5/Qsci/qsciglobal.h	2014-09-11 12:15:15.000000000 -0500
+++ src/QScintilla-gpl-2.8.4/Qt4Qt5/Qsci/qsciglobal.h	2015-04-28 12:28:11.208129000 -0500
@@ -40,18 +40,13 @@ extern "C++" {
 // Under Windows, define QSCINTILLA_MAKE_DLL to create a Scintilla DLL, or
 // define QSCINTILLA_DLL to link against a Scintilla DLL, or define neither
 // to either build or link against a static Scintilla library.
-#if defined(Q_OS_WIN)
 
-#if defined(QSCINTILLA_DLL)
-#define QSCINTILLA_EXPORT       __declspec(dllimport)
-#elif defined(QSCINTILLA_MAKE_DLL)
-#define QSCINTILLA_EXPORT       __declspec(dllexport)
-#endif
-
-#endif
-
-#if !defined(QSCINTILLA_EXPORT)
+#ifdef QT_STATIC
 #define QSCINTILLA_EXPORT
+#elif defined(QSCINTILLA_MAKE_DLL)
+#define QSCINTILLA_EXPORT       Q_DECL_EXPORT
+#else
+#define QSCINTILLA_EXPORT       Q_DECL_IMPORT
 #endif
 
 
