--- origsrc/QScintilla-gpl-2.8.4/Qt4Qt5/qscintilla.pro	2015-04-19 04:13:24.763841900 -0500
+++ src/QScintilla-gpl-2.8.4/Qt4Qt5/qscintilla.pro	2015-04-19 04:14:30.941637700 -0500
@@ -289,7 +289,7 @@ SOURCES = \
 	../lexers/LexInno.cpp \
 	../lexers/LexKix.cpp \
 	../lexers/LexKVIrc.cpp \
-	../lexers/LexLaTex.cpp \
+	../lexers/LexLaTeX.cpp \
 	../lexers/LexLisp.cpp \
 	../lexers/LexLout.cpp \
 	../lexers/LexLua.cpp \
